store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/FederatedStore.java

```diff
@@ -324,34 +324,50 @@ public List<String> getAllGraphIds(final User user, final boolean userRequesting
     }
 
     /**
-     * Get {@link Schema} for this FederatedStore
+     * This method exists for compatibility only. It will
+     * always return a blank {@link Schema}. Either use the
+     * {@link FederatedStore#getSchema} method and supply a
+     * {@link Context}, or ideally use the {@link GetSchema}
+     * operation instead.
      *
-     * @return schema
+     * @return {@link Schema} blank schema
      */
     @Override
     public Schema getSchema() {
-        return getSchema((Context) null);
+        return getSchema(new Context(), true);
```

Comment:
@GCHQDeveloper314 Has the default behaviour changed so that compact schema is now returned? Or was this always the behaviour?

Reply:
The original behaviour for this method was actually to almost always return an empty schema. Although that was not clear from this class (new Javadoc clarifies this). It still does this (though no longer via a deprecated class), so the **default behaviour is unchanged from before**.